test/Makefile

```diff
@@ -20,6 +20,7 @@ GTEST_DIR	= $(CURDIR)/../googletest/testing/googletest
 GMOCK_DIR	= $(CURDIR)/../googletest/testing/googlemock
 GTESTLIB_DIR	= $(CURDIR)/../googletest
 SNAPLIB_DIR	= $(CURDIR)/../lib
+PROTOBUF_DIR = /usr/local/lib
```

Comment:
What is that for? I think that this is standard library location used by linker and doesn't need to be passed.